vite.config.ts

```diff
@@ -0,0 +1,36 @@
+import { defineConfig } from 'vitest/config';
+import tsconfigPaths from 'vite-tsconfig-paths';
+import react from '@vitejs/plugin-react';
+import svgr from 'vite-plugin-svgr';
+import envCompatible from 'vite-plugin-env-compatible';
+
+const API_URL = process.env.UNLEASH_API || 'http://localhost:4242';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    test: {
+        globals: true,
+        setupFiles: 'src/setupTests.ts',
+        environment: 'jsdom',
+    },
+    server: {
+        proxy: {
+            '/api': {
```

Comment:
ok, so this is the part Ivar was worried about. Great that it works out of the box :]